src/ddbToEs/index.ts

```diff
@@ -30,15 +55,15 @@ export async function handleDdbToEsEvent(event: any) {
                 continue;
             }
 
-            const lowercaseResourceType = image.resourceType.toLowerCase();
+            const resourceType = tenantId ? `${tenantId}-${image.resourceType}` : image.resourceType;
```

Comment:
This is really no longer the resourceType
```suggestion
            const indexName= tenantId ? `${tenantId}-${image.resourceType}` : image.resourceType;
```